package.json

```diff
@@ -43,9 +43,9 @@
     "babelify": "browserify src/index.js -t [ babelify --presets [ @babel/preset-env ] --plugins [ @babel/plugin-proposal-object-rest-spread ] ] | uglifyjs > browser/gotrue.js",
     "prepare": "run-s compile",
     "prepublishOnly": "run-s prepublishOnly:*",
-    "prepublishOnly:checkout": "git checkout main",
-    "prepublishOnly:pull": "git pull",
-    "prepublishOnly:install": "npm ci",
+    "prepublishOnly:checkout": "run-local \"git checkout main\"",
```

Comment:
See https://github.com/netlify/eslint-config-node/blob/0807723152e6627274a13c3f93ced0f6c01c8bee/bin/run_local.js#L11

This keeps the option to run `npm publish` locally